tests/NPSP.py

```diff
@@ -42,10 +42,16 @@ def __init__(self, debug=False):
         self._init_locators()
 
     def _init_locators(self):
-        client = self.cumulusci.tooling
-        response = client._call_salesforce(
-            'GET', 'https://{}/services/data'.format(client.sf_instance))
-        latest_api_version = float(response.json()[-1]['version'])
+        try:
+            client = self.cumulusci.tooling
+            response = client._call_salesforce(
+                'GET', 'https://{}/services/data'.format(client.sf_instance))
+            latest_api_version = float(response.json()[-1]['version'])
+        except RobotNotRunningError:
+            # We aren't part of a running test, likely because we are
+            # generating keyword documentation. If that's the case, assume
+            # the latest supported version
+            latest_api_version = max(locators_by_api_version.keys())
```

Comment:
@boakley This seems like a good workaround. I've actually been intending to move the code for getting the API version from here into the CumulusCI or Salesforce library. Do you think we should do that now or separately?

Reply:
I think it should probably be a separate issue.

Reply:
I didn't put a ton of thought into this workaround. We can probably do better, but I didn't want that to derail getting the cci task finished.

Reply:
I created issue #W-031439 to create the function.